src/components/board/client.ts

```diff
@@ -1,12 +1,13 @@
 import { Struct } from 'google-protobuf/google/protobuf/struct_pb';
 
+import { Duration } from 'google-protobuf/google/protobuf/duration_pb';
 import { BoardServiceClient } from '../../gen/component/board/v1/board_pb_service';
 import type { RobotClient } from '../../robot';
 import type { Options, StructType } from '../../types';
 
 import pb from '../../gen/component/board/v1/board_pb';
 import { promisify, doCommandFromClient } from '../../utils';
-import type { Board } from './board';
+import type { Board, DurationOptions, PowerMode } from './board';
```

Comment:
[minor] see comment related to naming

```suggestion
import type { Board, Duration, PowerMode } from './board';
```